ledger/heavy/proc/send_code.go

```diff
@@ -35,32 +35,32 @@ type GetCode struct {
 	}
 }
 
-func NewGetCode(msg payload.Meta) *GetCode {
-	return &GetCode{
+func NewSendCode(msg payload.Meta) *SendCode {
+	return &SendCode{
 		message: msg,
 	}
 }
 
-func (p *GetCode) Proceed(ctx context.Context) error {
+func (p *SendCode) Proceed(ctx context.Context) error {
 	getCode := payload.GetCode{}
 	err := getCode.Unmarshal(p.message.Payload)
 	if err != nil {
-		return errors.Wrap(err, "failed to unmarshal GetCode message")
+		return errors.Wrap(err, "SendCode: failed to unmarshal GetCode message")
```

Comment:
This formatting is probably not necessary.

Reply:
This message concretise error. But here (where we have concrete `GetCode` type of message - maybe you right)